sympy/combinatorics/perm_groups.py

```diff
@@ -3448,6 +3510,11 @@ def _orbit_transversal(degree, generators, alpha, pairs, af=False):
     if ``af`` is ``True``, the transversal elements are given in
     array form.
 
+    If `slp` is `True`, a dictionary `{beta: slp_beta}` is returned
+    for `\beta \in Orb` where `slp_beta` is a list of indices of the
+    generators in `generators` s.t. if `slp = [i_1 ... i_n]`
+    `g_\beta = generators[i_1]*...*generators[i_n]`.
```

Comment:
The order should be checked. In the`*` product, the leftmost permutation is applied first.